tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -248,39 +238,6 @@ private void updateStatus(UpdateTicketRequest ticketRequest, RequestUtils reques
         ticketService.updateTicketStatus(ticket, ticketRequest.status(), requestUtils.toUserInstance());
     }
 
-    private void fileApprovalEntrySideEffects(FilesApprovalEntry ticket,
-                                              UpdateTicketRequest ticketRequest) throws ConflictException {
-
-        if (COMPLETED.equals(ticketRequest.status())) {
-            validateFilesForApproval(ticket);
-            ticket.approveFiles().persistUpdate(ticketService);
-        }
-    }
-
-    private void validateFilesForApproval(FilesApprovalEntry ticket) throws ConflictException {
-        if (filesForApprovalAreNotApprovable(ticket)) {
-            var fileIdsMissingMandatoryFields = getFileIdsMissingMandatoryFields(ticket);
-            throw new ConflictException(FILES_MISSING_MANDATORY_FIELDS_MESSAGE
-                                            .formatted(fileIdsMissingMandatoryFields));
-        }
-    }
-
-    private static String getFileIdsMissingMandatoryFields(FilesApprovalEntry ticket) {
-        return ticket.getFilesForApproval().stream()
-                   .map(PendingFile.class::cast)
-                   .filter(PendingFile::isNotApprovable)
-                   .map(File.class::cast)
-                   .map(File::getIdentifier)
-                   .map(String::valueOf)
-                   .collect(Collectors.joining(","));
-    }
-
-    private static boolean filesForApprovalAreNotApprovable(FilesApprovalEntry ticket) {
-        return ticket.getFilesForApproval().stream()
-                   .map(PendingFile.class::cast)
-                   .anyMatch(PendingFile::isNotApprovable);
-    }
```

Comment:
This check is also made in datamodel level when checking if file is approvable. Removing it from here, the only difference is that we will now serve bad request instead of conflict via API.